numpy/core/code_generators/ufunc_docstrings.py

```diff
@@ -1123,6 +1123,12 @@ def add_newdoc(place, name, doc):
     >>> np.equal(1, np.ones(1))
     array([ True])
 
+    If the inputs are ndarrays, then np.equal is equivalent to '=='
```

Comment:
I wonder if we want to invert this phrasing to something like

```suggestion
    The `==` operator on arrays is implemented using ``np.equal``
```

or


```suggestion
    The `==` operator on arrays is a shorthand for ``np.equal``
```

What I want to get across is that actually `np.equal` is the real deal, and `==` is a shorthand. Will defer to @bjnath and @rossbar who probably can come up with a better wording

Reply:
I haven't got any further comments about better wording so I'll wait on this for now. The PR is still in draft for now.

Reply:
I like @eric-wieser 's "implemented" suggestion, but either suggested wording is good IMO

Reply:
"Is a shorthand for" does sound better than "is implemented using". Does something along the lines of "The `==` operator can be used as a shorthand for``np.equal`` on ndarrays" sound ok?

Reply:
That sounds fine to me. @seberg, can you think of a wording that still leaves us open to adding a note that there are some cases when `==` is better than `equal` (due to what I remember as broken behavior on string/void arrays)?

Reply:
String/void is simply not implemented for `np.equal` (it would be possible, but annoying right now). There are some other (usually with deprecation warning), differences.  I thought about it for a while, and honestly do not have a good idea of how to try and achieve more precision. Unless we say "(see Notes for details)"?

Reply:
Have we reached a conclusion on this?

Reply:
It sounds to me like your latest wording suggestion ("The == operator can be used as a shorthand fornp.equal on ndarrays") is sufficient. Since there are no obvious technical problems with the wording, my vote would be to push ahead with the suggested wording. It can always be revisited for tweaks/improvements later on.